<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GPSSimProvider</class>
 <widget class="QWidget" name="GPSSimProvider">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>GPS Sim Provider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectPushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendSinglePushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Single Message</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendRepeatPushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Repeat Message</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
